server/src/main/java/org/tctalent/server/api/admin/CandidateBuilderSelector.java

```diff
@@ -190,29 +187,37 @@ private DtoBuilder candidateDto(
             .add("regoUtmMedium")
             .add("regoUtmSource")
             .add("regoUtmTerm")
-            .add("candidateExams", examsDto())
             .add("maxEducationLevel", educationLevelDto())
             .add("surveyType", surveyTypeDto())
             .add("country", countryDto())
             .add("nationality", countryDto())
-            .add("candidateOpportunities", candidateOpportunityDto())
             .add("user", userDto(userPropertyFilter))
             .add("candidateReviewStatusItems", reviewDto())
-
-            .add("candidateAttachments", candidateAttachmentDto(userPropertyFilter))
-            .add("shareableCv", candidateAttachmentDto(userPropertyFilter))
-            .add("shareableDoc", candidateAttachmentDto(userPropertyFilter))
-            .add("listShareableCv", candidateAttachmentDto(userPropertyFilter))
-            .add("listShareableDoc", candidateAttachmentDto(userPropertyFilter))
-            .add("taskAssignments", TaskDtoHelper.getTaskAssignmentDto())
-            .add("candidateProperties", candidatePropertyDto())
-            .add("shareableNotes")
-            .add("miniIntakeCompletedBy", userDto(userPropertyFilter))
+            .add("candidateAttachments", candidateAttachmentDto(userPropertyFilter, type))
             .add("miniIntakeCompletedDate")
-            .add("fullIntakeCompletedBy", userDto(userPropertyFilter))
             .add("fullIntakeCompletedDate")
-
             ;
+
+            if (!DtoType.PREVIEW.equals(type)) {
+                builder
+                    .add("candidateOpportunities", candidateOpportunityDto())
+                    .add("taskAssignments", TaskDtoHelper.getTaskAssignmentDto())
+                    .add("candidateExams", examsDto())
+                    .add("candidateProperties", candidatePropertyDto())
+                    .add("shareableCv", candidateAttachmentDto(userPropertyFilter, type))
+                    .add("shareableDoc", candidateAttachmentDto(userPropertyFilter, type))
+                    .add("listShareableCv", candidateAttachmentDto(userPropertyFilter, type))
+                    .add("listShareableDoc", candidateAttachmentDto(userPropertyFilter, type))
+                    .add("miniIntakeCompletedBy", userDto(userPropertyFilter))
+                    .add("fullIntakeCompletedBy", userDto(userPropertyFilter))
+                    .add("contextNote")
+                    .add("shareableNotes")
+                    .add("additionalInfo")
+                    .add("candidateMessage")
+                ;
+            }
+
+            return builder;
     }
```

Comment:
We both ended up tackling this one!

In fact the Preview of a candidate is more complex than this. There are some other of those related entities that have to be sent up with the preview because the data is used to display something in the summary line.
eg taskAssignments, candidateExams

See my issue #1284 , also in as a PR.

Reply:
Here I've merged changes from #1284 into this PR #1055. taskAssignment and candidateExams will be sent in previews; but also I think we'll need candidateReviewStatusItems for reviewable list previews - so leaving that in the preview DTO's as well.

Reply:
CandidateReviewStatus items are only related to searches, not lists